src/thrift.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+export const CLIENT_SEND = 'cs';
+export const CLIENT_RECV = 'cr';
+export const SERVER_SEND = 'ss';
+export const SERVER_RECV = 'sr';
+export const WIRE_SEND = 'ws';
+export const WIRE_RECV = 'wr';
+export const CLIENT_SEND_FRAGMENT = 'crf';
+export const SERVER_SEND_FRAGMENT = 'ssf';
+export const SERVER_RECV_FRAGMENT = 'srf';
```

Comment:
They should be removed i copied them from zipkin thrift file thinking we would need them.